crates/ra_ide_api/src/change.rs

```diff
@@ -235,7 +234,7 @@ impl RootDatabase {
 
     pub(crate) fn collect_garbage(&mut self) {
```

Comment:
I'd rather `if cfg!(feature = "wasm") { return }` instead of adding dependency on `instant`.

Reply:
The main issue is that we can't construct the RootDatabase without the `Instant::now()` for the `last_gc` fields, since that seems to be the only way to construct an `Ìnstant`.
Should we perhaps `#[cfg(not(feature = "wasm"))]` everything that references these fields?

Reply:
Hm, let’s add wasm_shims module p, which either reexports std::Instant or provides a no-op impl?

Reply:
Yeah, this works. There doesn't seem to be an issue with no-op impl.